.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,22 @@
 ## Proposed Commit Message
-<!-- Include a proposed commit message because all PRs are squash merged -->
+<!-- Include a proposed commit message because PRs are squash merged
+by default.
 
+See https://www.conventionalcommits.org/en/v1.0.0/#specification
+for our commit message convention.
+
+If the change is related to a particular cloud or particular distro,
+please include the "optional scope" in the summary line. E.g.,
+feat(ec2): Add support for foo to the baz
+
+Types used by this project:
+feat, fix, docs, ci, test, refactor, chore
+-->
 ```
-summary: no more than 70 characters
```

Comment:
I'd like to see us define an acceptable line LENGTH for titile/summary. I think it would be reasonable to set 72 as this width.

Reply:
Addressed in most recent force push